fiboa_cli/datasets/fieldscapes_austria_2021.py

```diff
@@ -0,0 +1,140 @@
+# TEMPLATE FOR A FIBOA CONVERTER
+#
+# Copy this file and rename it to something sensible.
+# The name of the file will be the name of the converter in the cli.
+# If you name it 'de_abc' you'll be able to run `fiboa convert de_abc` in the cli.
+
+from ..convert_utils import convert as convert_
+
+# File to read the data from
+# Can read any tabular data format that GeoPandas can read through read_file()
+# Supported protcols: HTTP(S), GCS, S3, or the local file system
+
+# Local URI added to the repository for initial conversion, Original Source https://beta.source.coop/esa/fusion-competition/
+URI = "/home/byteboogie/work/labwork_hkerner/fieldscapes/austria/boundaries_austria_2021.gpkg"
+
+# Unique identifier for the collection
+ID = "fieldscapes_austria_2021"
+# Title of the collection
+TITLE = "Field boundaries for Austria (Fieldscapes)"
+# Description of the collection. Can be multiline and include CommonMark.
+DESCRIPTION = """ The dataset contains field boundaries for the Austria."""
+# Bounding box of the data in WGS84 coordinates
+BBOX = [13.239974981742014, 48.204179578647796, 16.960943738443856, 48.974515524098045]
+
+# Provider name, can be None if not applicable, must be provided if PROVIDER_URL is provided
+PROVIDER_NAME = "Euro Crops"
+# URL to the homepage of the data or the provider, can be None if not applicable
+PROVIDER_URL = "https://data.europa.eu/data/datasets/ama_invekosreferenzensterreich2021?locale=en"
+# Attribution, can be None if not applicable
+ATTRIBUTION = "Publications Office of the European Union."
+
+# License of the data, either
+# 1. a SPDX license identifier (including "dl-de/by-2-0" / "dl-de/zero-2-0"), or
+LICENSE = "CC-BY-4.0"
+# 2. a STAC Link Object with relation type "license"
+# LICENSE = {"title": "CC-BY-4.0", "href": "https://creativecommons.org/licenses/by/4.0/", "type": "text/html", "rel": "license"}
+
+# Map original column names to fiboa property names
+# You also need to list any column that you may have added in the MIGRATION function (see below).
+COLUMNS = {
+    "FS_KENNUNG": "id",
+    "SL_FLAECHE": "area",
+    "EC_hcat_c": "crop_id",
+    "EC_hcat_n": "crop_name",
+    "geometry": "geometry"
+}
+
+# Add columns with constant values.
+# The key is the column name, the value is a constant value that's used for all rows.
+ADD_COLUMNS = {
+    "determination_datetime": "2021-01-01T00:00:00Z"
```

Comment:
Where does this information come from?`If it's unknown, better remove the line.

Reply:
I did a similar thing in the France dataset. These datasets from the government are released once a year, and so all the fields in it are clearly for that year. But they don't have any more specificity on a per field level. 

Putting in the first of the year is obviously the safest. I think with France I saw some 'release date', so I used that. 

It does seem very useful to know that it's from that year, so I lean towards including it in some way, like is done here.

Reply:
Yes, but yesterday we realized that people put things like the upload date ofthe source data there. That's not the determination time. 
We should start on the timestamps extension now, otherwise determination_datetime ends up being just whatever time the creator finds/has available.
As I've asked before in the meeting notes doc: Do you organize the meeting for the timestamps extension discussion or shall someone else take the lead? I can also organize it, but it sounded like you'd lead it, so don't want to interfere with you ;-) @cholmes

Reply:
> Yes, but yesterday we realized that people put things like the upload date ofthe source data there. That's not the determination time.

Agreed, that's not good. 

> Do you organize the meeting for the timestamps extension discussion or shall someone else take the lead? I can also organize it, but it sounded like you'd lead it, so don't want to interfere with you ;-) @cholmes

Yeah, sorry I've been negligent. I just sent out one on the deforestation regulation stuff. Will do a smaller timestamps / core one now.